### PR TITLE
[nginx-ingress] add helmfile for stable chart

### PR DESCRIPTION
**Pull request by alebabai**
## what
create helmfile for stable/nginx-ingress

## why
#19

### CODE REVIEW

File: helmfile.d/0330.stable-nginx-ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        resources:
+          limits:
+            cpu: "200m"
+            memory: "256Mi"
+          requests:
+            cpu: "50m"
+            memory: "128Mi"
+        service:
+          annotations:
+            ### Required: NGINX_INGRESS_HOSTNAME; e.g. ingress.us-west-2.cloudposse.org
```
Comment (aknysh) [inline helmfile.d/0330.stable-nginx-ingress.yaml]: Change `cloudposse.org` to `cloudposse.co`

File: helmfile.d/0330.stable-nginx-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            memory: "128Mi"
+        stats:
+          ### Optional: NGINX_INGRESS_CONTROLLER_STATS_ENABLED;
+          enabled: {{ env "NGINX_INGRESS_CONTROLLER_STATS_ENABLED" | default "true" }}
+        metrics:
+          ### Optional: NGINX_INGRESS_CONTROLLER_METRICS_ENABLED; enable Prometheus metrics (controller.stats.enabled must be true as well)
+          enabled: {{ env "NGINX_INGRESS_CONTROLLER_METRICS_ENABLED" | default "true" }}
+        service:
+          annotations:
+            ### Required: NGINX_INGRESS_HOSTNAME; e.g. ingress.us-west-2.cloudposse.org
```
Comment (aknysh) [inline helmfile.d/0330.stable-nginx-ingress.yaml]: Change `cloudposse.org` to `cloudposse.co`

**Review by aknysh — COMMENTED** [review]
LGTM
a few comments

**Review by osterman — APPROVED** [review]
LGTM

**Review by aknysh — APPROVED** [review]

**Pull request merged**